src/AudioSystem.cpp

```diff
@@ -1,14 +1,27 @@
 #include <iostream>
+#include <utility>
+#include <vector>
 
 #include "AudioSystem.h"
 
 #include "SDL.h"
 #include "SDL_mixer.h"
 
 namespace {
-    const int MIX_FREQ_HZ = 44100; // Sound sampling frequency in Hz. Change to "22050" if too slow.
-    const int MIX_NUM_CHANNELS = 2; // Number of sound channels for output
-    const int MIX_CHUNK_SIZE = 4096; // Bytes used per output sample
+    // Sound sampling frequency in Hz. Change to "22050" if too slow.
+    const int MIX_FREQ_HZ = 44100;
+    // Number of sound channels for output
+    const int MIX_NUM_CHANNELS = 2;
+    // Bytes used per output sample
+    const int MIX_CHUNK_SIZE = 4096;
+    // Information for loading sound assets. First value is the key, second is the path.
+    const std::vector<std::pair<const char*, const char*>> SOUND_ASSETS_INFO = {
```

Comment:
This should be an external config imo, doesn't have to be this pull request but it would use a YAML file with that library I linked before. I think the librayr I linked and ASSIMP serve different purposes.

Reply:
Yeah I agree with you here. I would definitely like to move this asset loading out of this system in the future. As it is now though is just the bare minimum for the audiosystem.